acceptance_tests/features/steps/create_message_internal.py

```diff
@@ -0,0 +1,102 @@
+from behave import given, when, then
+
+from acceptance_tests import browser
+from acceptance_tests.features.pages import create_message_internal
+
+
+@given("The internal user has found the associated respondent")
+def user_has_found_respondent(_):
+    assert create_message_internal.RESPONDENT_DETAILS.get_respondent_id() is not None
+    assert create_message_internal.RESPONDENT_DETAILS.get_ru_id() is not None
+    assert create_message_internal.RESPONDENT_DETAILS.get_ru_ref() is not None
```

Comment:
What is the difference between ru_id and ru_ref in this context?

Reply:
RU id's are uuid's used internally in the services, the RU ref is the reference number that the survey teams actually use to identify the RU's. They are 1:1 but both needed for different purposes.